test/lib/queue_worker.spec.js

```diff
@@ -2117,7 +2117,7 @@ describe('QueueWorker', function() {
         setTimeout(function() {
           callbackComplete = true;
           resolve();
-        }, 250);
+        }, 100);
```

Comment:
2120: No

2150: Yes (On my workstation expect(callbackStarted).to.be.true fails because the queue hasn't picked up the task yet.

I will fix 2120 it was an artifact of fiddling around with it.